### ISSUE TITLE
Add details about quick actions keyboard shortcut

### ISSUE THREAD

**Issue by Liversage** [MicrosoftDocs/visualstudio-docs#3164]
The article explains that `Ctrl+.` is the keyboard shortcut to apply a quick action. The screenshot also shows a tooltip with the text __Show potential fixes__ followed by the keyboard shortcuts that can be used to apply the quick action.

However, on my fresh installation of Visual Studio 2019  `Ctrl+.` did not apply the quick action, nor was there any keyboard shortcuts listed in the tooltip.

After much searching and guesswork I found that the keyboard command to bind is `View.QuickActionsForPosition`. In my case this command was bound to `Ctrl+½, Ctrl+.`

I suggest that whenever you document a keyboard shortcut that you also name the specific command that the shortcut invokes to allow people with other keyboard bindings to easily rebind the command.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 93c0395a-7b86-46ca-9ede-f7cf361f2844
* Version Independent ID: 2ef2a75f-77e5-0192-3a64-a555178f02e3
* Content: [Quick Actions, light bulbs, and screwdrivers - Visual Studio](https://docs.microsoft.com/en-us/visualstudio/ide/quick-actions?view=vs-2019#feedback)
* Content Source: [docs/ide/quick-actions.md](https://github.com/MicrosoftDocs/visualstudio-docs/blob/master/docs/ide/quick-actions.md)
* Product: **visual-studio-windows**
* Technology: **vs-ide-general**
* GitHub Login: @gewarren
* Microsoft Alias: **gewarren**

**Comment by gewarren** [comment]
@Liversage Thanks, that is actually great feedback. We need to go through all of our docs and make sure to either say which profile the keyboard shortcut is for, or what the command is, as you mentioned.

**Comment by gewarren** [comment]
Edit to trigger priority labeling.

**Comment by TerryGLee** [comment]
@Liversage Apologies that it's taken us a while to circle back with you on this issue. I've taken a look at the settings for Quick Actions in the current-as-of-today's-date, [version 16.8.4](https://docs.microsoft.com/en-us/visualstudio/releases/2019/release-notes?view=vs-2019#16.8.4) of Visual Studio 2019, and noted the following:

![image](https://user-images.githubusercontent.com/23244653/105224453-0f599480-5b12-11eb-9810-72e1007840c9.png)

We've also got a page posted, [**Default keyboard shortcuts in Visual Studio**](https://docs.microsoft.com/en-us/visualstudio/ide/default-keyboard-shortcuts-in-visual-studio?view=vs-2019), which lists most of the keyboard shortcuts and their associated commands, along with instructions on how to change them. We'll add that link to this page, so if others run in to the problem you identified, they can troubleshoot it more easily.

Additionally, we'll take our colleague's suggestion under advisement, i.e. to comb through our docs and "make sure to either say which profile the keyboard shortcut is for, or what the command is, as you mentioned." We've added that task to our backlog queue.

Meanwhile, we'll close this issue, with our thanks to you for helping us to improve the documentation!
